views/download_fluent_package.erb

```diff
@@ -23,6 +23,153 @@
 
   <div class="row margin-bottom-20">
     <div class="col-md-9">
+
+      <h4><a name=fluent-package></a>Fluent Package (fluent-package)</h4>
+      <p style='color: red'>fluent-package v5.0.0 is a RC (Release Candidate) version of fluent-package v5 series. Use 5.0.1 or later - GA (General Availability) version of v5 series.</p>
```

Comment:
> fluent-package v5.0.0 is a RC (Release Candidate) version of fluent-package v5 series. Use 5.0.1 or later - GA (General Availability) version of v5 series.

I wonder if this caution is unnecessary since it is quite usual to choose the version of the latest patch version.

Instead of this caution, how about adding an explanation about this package and the channels.

```suggestion
      <p>Fluent Package (fluent-package), formerly known as Treasure Agent (td-agent). There are two channels:</p>
      <ul>
        <li>Normal release</li>
        <li>Long Term Support (LTS)</li>
      </ul>
      <p>We ship Normal release in rapid release development style. You can use the latest feature of Fluentd.</p>
      <p>On the other, Long Term Support (LTS) is a more conservative maintenance version which will not introduce a new feature. It only applies teeny updates such as security or bug fixes. We recommend the LTS channel for enterprise services.</p>
      <p>* LTS for v5 will be supported until March, 2025. Please see <a href="/blog/fluent-package-scheduled-lifecycle">fluent-package-scheduled-lifecycle</a> for more details.</p>
```